### PR TITLE
Add a desktop version

### PR DESCRIPTION
**Pull request by NickEmma**

### CODE REVIEW

**Comment by NickEmma** [comment]
## Desktop version design
- Added a background image
- Added a grid to layout to cards vertically side by side
- Added the contact to layout side by side

File: index.html [hunk 1]
```diff
@@ -336,44 +342,52 @@ <h3>Skills</h3>
 
     <section class="contact">
```
Comment (tufoinnkuo10) [inline index.html:343]: - [ ] please make your about section cards to be more visible. 
Screenshot (107)_LI

File: index.html [hunk 2]
```diff
@@ -6,12 +6,18 @@
...
   <body>
     <header class="header">
       <a href="#" class="logo">techie</a>
       <nav class="navbar">
         <img src="images/Enabled.png" class="menu" alt="menu-icon" />
+        <ul>
+          <li><a href="#">portfolio</a></li>
+          <li><a href="#">about</a></li>
+          <li><a href="#">contact</a></li>
+        </ul>
```
[the remark this answers is not in the document]
Comment (NickEmma) [reply index.html:20]: @tufoinnkuo10  Cool thanks for the reminder.

File: index.html (hunk 1 above)
Comment (NickEmma) [reply index.html:343]: cool thanks for that also

**Pull request merged**